assets/ajax-cart.js.liquid

```diff
@@ -299,7 +299,11 @@ var ajaxCart = (function(module, $) {
         itemMinus: cartItem.quantity - 1,
         itemQty: cartItem.quantity,
         price: Shopify.formatMoney(cartItem.price, settings.moneyFormat),
-        vendor: cartItem.vendor
+        vendor: cartItem.vendor,
+        linePrice: Shopify.formatMoney(cartItem.line_price, settings.moneyFormat),
+        originalPrice: Shopify.formatMoney((cartItem.line_price + cartItem.total_discount), settings.moneyFormat), 
```

Comment:
Since this is talking about `cartItem.line_price`, this can be:

```
originalLinePrice: Shopify.formatMoney(cartItem.original_line_price, settings.moneyFormat)
```